Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -4137,6 +4137,24 @@ void HTMLMediaElement::pauseInternal()
     updatePlayState();
 }
 
+bool HTMLMediaElement::hasMediaSource() const
+{
+#if ENABLE(MEDIA_SOURCE)
+    if (m_mediaSource)
+        return true;
+#endif
+    return false;
+}
+
+bool HTMLMediaElement::hasManagedMediaSource() const
+{
+#if ENABLE(MANAGED_MEDIA_SOURCE)
```

Comment:
Should probably be:
```
#if ENABLE(MANAGED_MEDIA_SOURCE)
    return is<ManagedMediaSource>(m_mediaSource);
#else
    return false;
#endif
```

or

```
#if ENABLE(MANAGED_MEDIA_SOURCE)
    if (m_mediaSource)
        return is<ManagedMediaSource>(*m_mediaSource);
#endif
    return false;
```

Reply:
`is<>` already does a null check for you when you give it a pointer instead of a reference.